airbyte-cdk/python/airbyte_cdk/utils/airbyte_secrets_utils.py

```diff
@@ -2,24 +2,43 @@
 # Copyright (c) 2022 Airbyte, Inc., all rights reserved.
 #
 
-import logging
-from typing import TYPE_CHECKING, Any, List, Mapping
+from typing import Any, List, Mapping
 
-from airbyte_cdk.utils.mapping_utils import all_key_pairs_dot_notation, get_value_by_dot_notation
+import dpath.util
 
-if TYPE_CHECKING:
-    from airbyte_cdk.sources import Source
 
+def get_secret_paths(schema: Mapping[str, Any]) -> List[str]:
+    paths = []
 
-def get_secrets(source: "Source", config: Mapping[str, Any], logger: logging.Logger) -> List[Any]:
+    def traverse_schema(schema: Any, path: List[str]):
```

Comment:
adapted from schema_helpers and now uses nested lists to indicate the path in the JSON instead of `/` separators